### PR TITLE
system view issues

### PR DESCRIPTION
**Pull request by sutcalag**
#262 

Create / update time have api, but no value
Refine chart click behaviour
Should chose first node in node list view by default
Some data in system configuration is not shown up. Like: min_segment_size_to_enable_index, minio_bucket_name.. each coord should have different configuration.
Logout not working for system view

### CODE REVIEW

File: client/src/plugins/system/DataCard.tsx [hunk 1]
```diff
@@ -137,9 +137,23 @@ const DataCard: FC<DataCardProps & React.HTMLAttributes<HTMLDivElement>> = (prop
...
+
+  const configTitle = [t('configTitle'), t('valueTitle')];
+  const systemConfig: { label: string; value: any; }[] = [];
+
+  const systemTitle = [t('systemTitle'), t('valueTitle')];
+  const systemContent = [];
+
+  const {
+    created_time: create,
+    updated_time: update,
```
Comment (nameczz) [inline client/src/plugins/system/DataCard.tsx:152]: createdTime / updatedTime is more clear to me

File: client/src/plugins/system/NodeListView.tsx [hunk 2]
```diff
@@ -124,6 +123,16 @@ const NodeListView: FC<NodeListViewProps> = (props) => {
     }
   }, [selectedCord, childNodes, capacityTrans]);
 
+  useEffect(() => {
+    const timeoutID = window.setTimeout(() => {
```
Comment (nameczz) [inline client/src/plugins/system/NodeListView.tsx:127]: what's this for? need some comments

File: client/src/plugins/system/DataCard.tsx [hunk 3]
```diff
@@ -163,19 +177,20 @@ const DataCard: FC<DataCardProps & React.HTMLAttributes<HTMLDivElement>> = (prop
     });
   }
 
-  const systemTitle = [t('systemTitle'), t('valueTitle')];
-  const systemContent = [];
-  const sysInfos = node?.infos?.system_info || {};
+  if (system_configurations) {
+    Object.keys(system_configurations).forEach(key => {
+      systemConfig.push({ label: key, value: system_configurations[key] });
```
Comment (nameczz) [inline client/src/plugins/system/DataCard.tsx:182]: In this components, why we don't use useEffect to do data transform? It  looks like all data transform operations running every render .

**Review by nameczz — CHANGES_REQUESTED** [review]

File: client/src/plugins/system/DataCard.tsx (hunk 3 above)
Comment (sutcalag) [reply client/src/plugins/system/DataCard.tsx:182]: This component is a pure component without any user action. So re-render means data changes.

**Review by nameczz — APPROVED** [review]

**Review by shanghaikid — APPROVED** [review]

**Review by Tumao727 — APPROVED** [review]

**Pull request merged**